Packs/ServiceNow/Integrations/ServiceNowEventCollector/README.md

```diff
@@ -1,37 +1,33 @@
-Use this integration to fetch audit logs from ServiceNow as Cortex XSIAM events.
+Use this integration to fetch audit and syslog transactions logs from ServiceNow as Cortex XSIAM events.
 This integration was integrated and tested with Vancouver version of ServiceNow API.
 
-## Configure ServiceNow Event Collector on Cortex XSOAR
+## Configure ServiceNow Event Collector in Cortex
 
```

Comment:
1. Navigate to **Settings** > **Configurations** > **Data Collection** > **Automations & Feed Integrations**.

2. Search for ServiceNow Event Collector.

3. Click **Add instance** to create and configure a new integration instance.